### PR TITLE
CLOUDP-67926: mongocli iam organizations|orgs apiKey(s)|apikey(s) describe <id>

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-67926

Closes #[issue number]

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli iam org apikeys describe 5f2144a8ee43f61c6e42ffba -o ""
ID				DESCRIPTION	PUBLIC KEY	PRIVATE KEY
5f2144a8ee43f61c6e42ffba	Test		dnfesree	********-****-****-27476b125f6e
```

```
./bin/mongocli iam org apikeys describe 5f2144a8ee43f61c6e42ffba
{
  "id": "5f2144a8ee43f61c6e42ffba",
  "desc": "Test",
  "roles": [
    {
      "groupId": "5e4e593f70dfbf1010295836",
      "roleName": "GROUP_DATA_ACCESS_READ_WRITE"
    },
    {
      "groupId": "5e4e593f70dfbf1010295836",
      "roleName": "GROUP_CLUSTER_MANAGER"
    },
    {
      "groupId": "5e4e593f70dfbf1010295836",
      "roleName": "GROUP_DATA_ACCESS_ADMIN"
    },
    {
      "groupId": "5e4e593f70dfbf1010295836",
      "roleName": "GROUP_OWNER"
    },
    {
      "groupId": "5e4e593f70dfbf1010295836",
      "roleName": "GROUP_READ_ONLY"
    },
    {
      "orgId": "5e429e7706822c6eac4d5970",
      "roleName": "ORG_MEMBER"
    },
    {
      "groupId": "5e4e593f70dfbf1010295836",
      "roleName": "GROUP_DATA_ACCESS_READ_ONLY"
    }
  ],
  "privateKey": "********-****-****-27476b125f6e",
  "publicKey": "dnfesree"
}
```

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
Waiting for #315

File: internal/description/description.go [hunk 1]
```diff
@@ -14,139 +14,140 @@
...
+	ConfigDescription           = "Configure a profile to store access settings for your MongoDB deployment."
+	ConfigSetDescription        = "Configure specific properties of a profile."
+	ConfigRenameDescription     = "Rename a profile."
+	ConfigDeleteDescription     = "Delete a profile."
+	ConfigList                  = "List available profiles."
+	ConfigDescribe              = "Return a specific profile"
+	IAM                         = "Organization and projects operations."
+	Organization                = "Organization operations."
+	APIKeys                     = "APIKeys operations."
+	ListOrganizationAPIKeys     = "List all Organization API Keys associated with {ORG-ID}."
```
Comment (gssbzn) [inline internal/description/description.go:117]:
```suggestion
	ListOrganizationAPIKeys     = "List all organization API Keys for a given organization."
```

File: internal/store/api_keys.go [hunk 2]
```diff
@@ -42,3 +46,17 @@ func (s *Store) APIKeys(orgID string, opts *atlas.ListOptions) ([]atlas.APIKey,
 		return nil, fmt.Errorf("unsupported service: %s", s.service)
 	}
 }
+
+// APIKey encapsulate the logic to manage different cloud providers
```
Comment (robcarlan-mlab) [inline internal/store/api_keys.go:50]: copy/paste comment here

File: internal/store/api_keys.go [hunk 3]
```diff
@@ -42,3 +46,17 @@ func (s *Store) APIKeys(orgID string, opts *atlas.ListOptions) ([]atlas.APIKey,
...
+func (s *Store) APIKey(orgID, apiKeyID string) (*atlas.APIKey, error) {
```
Comment (robcarlan-mlab) [inline internal/store/api_keys.go:51]: I'll probably add the organization project keys here too -- are you ok if I change these in my pr to prefix with `Project`?

**Comment by andreaangiolillo** [comment]
I am not sure how to retry the evergreen build on github 😅

**Comment by andreaangiolillo** [comment]
evergreen retry

**Comment by andreaangiolillo** [comment]
"evergreen retry"

**Comment by andreaangiolillo** [comment]
“evergreen retry”

**Comment by andreaangiolillo** [comment]
evergreen retry

**Pull request merged**